tests/units/Services/Table/BuilderTest.php

```diff
@@ -280,24 +333,24 @@ private function createTestModelTable()
             $table->timestamps();
         });
     }
-
 }
 
-
 class BuilderTestModel extends Model
 {
-    protected $fillable = ['name', 'is_active', 'price'];
+    protected $fillable = ['name', 'price', 'is_active'];
+
+    protected $casts = ['is_active' => 'boolean'];
```

Comment:
I want to ask you a lot of questions :). because I want to adhere to them and I want to make a mistake again.
What is your rule for sorting attribute?

Reply:
id, fk, main attributes (name, code etc), additional attributes, booleans, created_by / updated_by, timestamps.

I try to keep the same order in migrations, fillable, request validators, resources etc